moped-database/metadata/tables.yaml

```diff
@@ -5493,6 +5493,9 @@
           - workgroup_name
         filter: {}
         allow_aggregations: true
+- table:
+    name: project_funding_view
+    schema: public
```

Comment:
Yea - I dig this going into the metadata, and not because I like to use the console as a client -- y'all know it's not my fav -- but I like it rather because this is now an explicit declaration that we don't offer this data up over the graphql endpoint.

![image](https://github.com/user-attachments/assets/80b3f10a-2e69-4809-b070-675e3cb333bb)